.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "start",
+			"problemMatcher": [],
+			"label": "npm: start",
+			"detail": "npm run dev"
+		}
+	]
+}
```

Comment:
I not convinced that this file should be added to the scaffold. I'd be included to delete it an add it to gitignore, but if it is to be added then it should needs to list all of the npm scripts with accurate names. If you think we should have this then it could be added under a different issue/PR?

Reply:
Absolutely. Removed.